include/lo2s/bfd_resolve.hpp

```diff
@@ -107,6 +107,26 @@ class LookupError : public std::runtime_error
     }
 };
 
+struct bfd_handle_deleter
+{
+    void operator()(bfd* p) const
+    {
+        bfd_close(p);
+        p = nullptr;
```

Comment:
This assignment only affects the local `p`, which is passed-by-value into this function. So no need for that.